spec/rspec/rails/matchers/active_job_spec.rb

```diff
@@ -144,5 +169,41 @@ def perform; end
         }.to have_enqueued_job.at_most(:once)
       }.to raise_error(/expected to enqueue at most 1 jobs, but enqueued 2/)
     end
+
+    it "passes with provided queue name" do
+      expect {
+        hello_job.set(:queue => "low").perform_later
+      }.to have_enqueued_job.on_queue("low")
+    end
+
+    it "passes with provided at date" do
+      expect {
+        hello_job.set(:wait_until => Date.tomorrow.noon).perform_later
```

Comment:
Can you extract `Date.tomorrow.noon` into a variable? I know the window is small, but it's possible for this to fail when run precisely at midnight when the first call is before the date change and the second call is after.